### PR TITLE
automaticly stop resources that are breaking esx

### PR DESCRIPTION
**Pull request by Arctos2win**

### CODE REVIEW

**Comment by Kenshiin13** [comment]
Looks good. Though, I'd appreciate if you formated your strings instead of concatenating them for the sake of readability.

File: [core]/es_extended/server/main.lua [hunk 1]
```diff
@@ -701,3 +701,28 @@ end)
...
+    ['fivem-map-skater'] = true,
+    ['fivem-map-hipster'] = true,
+    ['qb-core'] = true,
+    ['default_spawnpoint'] = true,
+}
+
+for key in pairs(DoNotUse) do 
+    if GetResourceState(key) == 'started' or GetResourceState(key) == 'starting' then
+		StopResource(key)
+        print("[^1ERROR^7] WE STOPPED A RESOURCE THAT WILL BREAK ^1ESX^7, PLEASE REMOVE ^5" .. key .. "^7")
```
Comment (Gellipapa) [inline [core]/es_extended/server/main.lua:719]: Hi! Please use ():format.

File: [core]/es_extended/server/main.lua [hunk 2]
```diff
@@ -701,3 +701,28 @@ end)
...
+    end
+end
+
+AddEventHandler('onResourceStart', function(resourceName)
+    if DoNotUse[string.lower(resourceName)] then 
+		StopResource(key)
+        print("[^1ERROR^7] WE STOPPED A RESOURCE THAT WILL BREAK ^1ESX^7, PLEASE REMOVE ^5" .. resourceName .. "^7")
```
Comment (Gellipapa) [inline [core]/es_extended/server/main.lua:726]: Hi! Please use ():format.

**Review by Gellipapa — CHANGES_REQUESTED** [review]

**Comment by Gellipapa** [comment]
Hi! Well done, I'll test and merge.

**Review by Gellipapa — APPROVED** [review]

**Pull request merged**